main.cc

```diff
@@ -48,7 +49,9 @@ ostream& operator<<(ostream& out, const input_paras& ip) {
       << "st_start_prog:" << ip.st_start_prog << endl
       << "restart_w_e_list:" << ip.restart_w_e_list << endl
       << "restart_w_p_list:" << ip.restart_w_p_list << endl
-      << "reset_win_niter:" << ip.reset_win_niter << endl;
+      //<< "restart_w_s_list:" << ip.restart_w_s_list << endl
```

Comment:
`restart_w_e_list` stores the error costs used for restarting a search procedure.
For example, restart_w_e_list = {1, 2}
We use a round-robin fashion. w_e is 1 for the first search. After restarting the search, w_e is set as 2. After restarting the search, w_e is set as 1, ...
Could you please support `restart_w_s_list` for restarting?